test/playwright/utils/navigation.ts

```diff
@@ -12,7 +12,8 @@ import {
 import { SCREEN_SIZES } from '~/constants/screens'
 
 import enMessages from '~/locales/en.json'
-import rtlMessages from '~/locales/ar.json'
+
+import rtlMessages from '../../locales/ar.json'
```

Comment:
```suggestion
import rtlMessages from '~~/test/locales/ar.json'
```

In case we want to avoid relative imports.

Reply:
We do, so I'm going to commit this :)